examples/live-agent/src/main.tsx

```diff
@@ -0,0 +1,10 @@
+import { ChatWidget } from '@voiceflow/react-chat';
+import { createRoot } from 'react-dom/client';
+
+import { RuntimeProvider } from './context';
+
+createRoot(document.getElementById('root')!).render(
+  <RuntimeProvider>
+    <ChatWidget chatAPI={undefined} />
```

Comment:
Hey, guys! @ninabondar could you take a look into why is this undefined? What is this supposed to be?